fusor/fusor.py

```diff
@@ -549,12 +551,16 @@ def _location_descriptor(
             Descriptor's id. `False`, use default of `label` > `sequence_id`
         """
         seq_id_input = sequence_id
+
         try:
             sequence_id = coerce_namespace(sequence_id)
         except ValueError:
-            try:
-                CURIE(__root__=sequence_id)
-            except ValidationError:
+            # following no longer throws validation error after v2 change??
```

Comment:
commenting for eyes here - it looks like this should still throw the `ValidationError` bc that's what `constr(pattern....` should do if the regex doesn't match. But this line would successfully make a `CURIE` and never throw the exception. Any ideas? Is the `if` I added below an okay solution to this?

Reply:
@korikuzma @jsstevenson I think I've got everything addressed and up to date on this PR 😄 Just wanted to tag y'all here and get your thoughts on this code/to make sure you saw it!